tailnet/controllers.go

```diff
@@ -1150,19 +1237,25 @@ func (t *tunnelUpdater) allAgentIDs() []uuid.UUID {
 	return out
 }
 
-func (t *tunnelUpdater) allDNSNames() map[dnsname.FQDN][]netip.Addr {
+// updateDNSNames updates the DNS names for all workspaces in the tunnelUpdater.
+func (t *tunnelUpdater) updateDNSNames() map[dnsname.FQDN][]netip.Addr {
 	names := make(map[dnsname.FQDN][]netip.Addr)
 	for _, w := range t.workspaces {
-		err := w.addAllDNSNames(names, t.ownerUsername)
```

Comment:
I moved the `ownerUsername` from an argument when updating a workspace to a private field on the `Workspace`, as a form of future-proofing.